modules/cms/classes/Theme.php

```diff
@@ -402,25 +420,63 @@ public function getFormConfig()
         return $config;
     }
 
+    /**
+     * Generates an asset URL for the provided path within the theme, will use the parent theme
+     * if the current theme does not actually have a directory on the filesystem (i.e. is virtual).
+     */
+    public function assetUrl(?string $path): string
+    {
+        $cacheTime = Config::get('cms.urlCacheTtl', 10);
```

Comment:
Cache expiration is set in seconds, not minutes, best practice is to always do `now()->addMinutes(10)` instead for most explicit use

Reply:
We should probably update this comment then
https://github.com/wintercms/winter/blob/ffa6f99210234f86891def675df69341d30f925c/config/cms.php#L172

And change this value
https://github.com/wintercms/winter/blob/ffa6f99210234f86891def675df69341d30f925c/config/cms.php#L176

Reply:
Added this in [1a8fb44](https://github.com/wintercms/winter/pull/726/commits/1a8fb449a9f9989f19ce986b6e442c29cac03ece)